docs/sites.yml

```diff
@@ -6861,3 +6861,19 @@
   built_by: 97thjingba
   built_by_url: https://github.com/97thjingba
   featured: false
+- title: George
+  main_url: https://kind-mestorf-5a2bc0.netlify.com
+  url: https://kind-mestorf-5a2bc0.netlify.com
+  description: >
+    shiny new web built with Gatsby
+  categories:
+    - Blog
+    - Portfolio
+    - Gallery
+    - Landing page
```

Comment:
```suggestion
```

Valid values are `API, Accessibility, Agency, App, Blog, Business, Community, Conference, Consulting, Data, Design, Design System, Directory, Documentation, Education, Entertainment, Entrepreneurship, Event, Featured, Finance, Food, Freelance, Gallery, Government, Healthcare, JavaScript, Landing Page, Learning, Library, Marketing, Media, Mobile Development, Music, Nonprofit, Open Source, Photography, Podcast, Portfolio, Productivity, Programming, Real Estate, Retail, SEO, Science, Security, Sports, Technology, Travel, User Experience, Video, Web Development, WordPress, eCommerce`

Reply:
tho, what was wrong with other prev tags/categories? Ok nevermind

Reply:
Nothing wrong per se! 🙂 We just have these checks in place (and the whitelist) to prevent too many _one off_ category tags. Tags are only as useful as they are used consistently! 😉 

We could add `Landing page` to the whitelist but it seems pretty generic so I would prefer if we removed it. What do you think?

Reply:
@EarlGeorge Can you remove it please?

Reply:
hm, tbh not a big fun of leaving without it. u don't like my landing page? :(

Reply:
We love your landing page! 🤗 But we just don't have any other sites with the tag of `Landing page`. That's why it's not a whitelisted category!

Reply:
noway, are u kidding how about this one https://www.gatsbyjs.org/showcase/?filters%5B0%5D=Landing%20Page

Reply:
Aha! It was the case sensitivity 😂 

`Landing page` needs to be `Landing Page`

Sorry I didn't notice that!

Reply:
it's insane maybe you haven't noticed my other initial tags as well that u have deleted? 😂

Reply:
What other initial tags have I deleted? We have a test for this in https://github.com/gatsbyjs/gatsby/pull/16409/checks

This is _indeed_ incorrect (needs to be `Landing Page` not `Landing page`). Can you fix the casing so I can merge this?

Reply:
ok

Reply:
```suggestion
    - Landing Page
```

Reply:
@EarlGeorge Can you please hit `commit suggestion`?

Reply:
we gucci 😂